_components/sidenav/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/side-navigation/accessibility-tests/
+layout: accessibility-test
+component:
+  name: side navigation
+title: Side navigation accessibility tests
+category: Components
+lead: Any USWDS [component name] component should pass these manual accessibility tests.
```

Comment:
```suggestion
lead: Any USWDS side navigation component should pass these manual accessibility tests.
```

Reply:
Resolved in [9c30372](https://github.com/uswds/uswds-site/pull/2864/commits/9c3037216def30ecfc367e0adf4292a569722918)